docs/DeveloperGuide.md

```diff
@@ -167,13 +167,51 @@ Classes used by multiple components are in the `seedu.taassist.commons` package.
 
 ## **Implementation**
 
+### Creating and deleting classes
+
+Class information is stored as `ModuleClass` objects, which captures the name of the class as well as the sessions created for the class. When the user creates a class, the program creates a new `ModuleClass` object with the class name and adds it to the collection of `ModuleClass` objects previously created, which is managed by the `TaAssist` class.
+
+When deleting classes, all students previously assigned to those classes will have the classes unassigned from them, before the `ModuleClass` objects corresponding to those classes are removed from the collection captured by `TaAssist`.
```

Comment:
I think `module classes` will sound less confusing than `classes` cos `classes` can mean Java classes. Same for other parts of the DG.